File: AST/output/program actual outputs/P Progs/isPrimeNumber.py
import sys

def main():
	n, c = 0, 2

	print("Enter an integer\n")

	n = input()

	if(n == 2):
		print("Prime number.\n")


	else:
		for c in range(2, n):
			if(n % c == 0):
				break




		if(c != n):
			print("Not prime.\n")


		else:
			print("Prime number.\n")






	return 

File: AST/output/program actual outputs/C Progs/isPrimeNumber.c
#include <stdio.h>

int main(){
	int n, c;

	printf("Enter an integer\n");

	scanf("%d", &n);

	if(n == 2) {
		printf("Prime number.\n");
	}
	else {
		for (c = 2;c <= n - 1;c++){
			if(n % c == 0) {
				break;
			}


		}
		if(c != n) {
			printf("Not prime.\n");
		}
		else {
			printf("Prime number.\n");
		}

	}


	return 0;

}

